docs/modules/operation/pages/flows/introduction.adoc

```diff
@@ -1,22 +1,41 @@
 
 [[ga-flow-support-introduction]]
-= Flow Support
+= Flows
 
-{page-component-title} supports receiving, decoding, and persisting flow information sent from your network devices.
-A list of currently supported protocols is available in the <<reference:telemetryd/protocols/introduction.adoc#ref-protocol, Telemetry>> section.
-While flows offer a great breadth of information, the current focus of the support in {page-component-title} is aimed at:
+Flows refers to the summary of network traffic sent by network devices (switches, routers, and so on).
+This information includes, but is not limited to, source and destination address, source and destination port, octet count, and duration of activity.
+Collecting and analyzing flows data provides a picture of network usage and helps to diagnose network issues.
+Persisting flows for long-term storage can aid in forensic analysis.
 
-* Network diagnostic: viewing the top protocols and top talkers within the context of a particular network interface.
-* Forensic analysis: persisting the flows for long-term storage.
+{page-component-title} provides the following:
+
+* A platform to collect, persist, and visualize flows, with support for NetFlow versions 5 and 9, IPFIX, and sFlow
+* Inventory enrichment (mapping to OpenNMS nodes)
+* Application classification
+* Horizontal scaling (tested with 250,000+ flows/second)
```

Comment:
I would remove the 250.000+ flows/second. It's more something for a data sheet and as soon we do a different number no one will remember to update this place here.